### PR TITLE
Make touch layout position accurate

### LINKED ISSUE #14743: Touch Control customization screen is super buggy
The Touch Controls customization screen is a buggy mess and it's impossible to use:
1. You can move and resize **hidden buttons**: sometimes you may notice that a button you're dragging is not moving at all. It's because a hidden button overlapping with it is moving instead!

2. The edit area doesn't match the actual in-game area: How is one supposed to know if the control setup is right when the edit area doesn't even match the actual in-game controls?! 
Solution: remove the side bar and put it at the top instead. Almost no one uses that area. Make it super small, and preferably even hideable.

3. You can't put controls off the edit area: Believe it or not, some people like to put the controls at the edge of the screen. It's impossible to do rn. You should only make sure that the control doesn't go completely off-screen (at least one vertex must be inside the edit area in a way that it can be dragged)

Bugs 1 and 3 are easy to fix. But 2 may need some deep changes in UI.

### PR DESCRIPTION
**Pull request by iota97**
This make screen rateo and scale correct. Fixes #14743.

I prefer this rather than add a drag drop menu or such for the left column button as I think that will be really hard to get the UX right (like if I place a button next to the edge and I want to drag it back should I open the menu or no?)

Result:
Screenshot_2021-08-20_12-44-41

This is just a test to show it's accurate (ofc the column width is normal in the PR):
Screenshot_2021-08-20_12-43-51
Screenshot_2021-08-20_12-43-36

### CODE REVIEW

**Comment by hrydgard** [comment]
Hm, the arrow keys look weirdly large in the picture where you prove that it's accurate?

**Comment by iota97** [comment]
Oh yeah, its just little spacing and big scale, was to check if the line overlap in the same way. Was just to test with scale and spacing as well, not a real layout I would use for sure.

File: UI/TouchControlLayoutScreen.cpp [hunk 1]
```diff
@@ -612,7 +619,8 @@ void TouchControlLayoutScreen::CreateViews() {
 	tabHolder->SetTag("TouchControlLayout");
 	root_->Add(tabHolder);
 
-	layoutView_ = root_->Add(new ControlLayoutView(new AnchorLayoutParams(leftColumnWidth + 10, 0.0f, 0.0f, 0.0f, false)));
+	root_->Add(new ItemHeader("", new AnchorLayoutParams(leftColumnWidth + 10, bounds.h*(1.0-layoutAreaScale)-40, NONE, NONE, false)));
```
Comment (unknownbrackets) [inline UI/TouchControlLayoutScreen.cpp:622]: Hm, it's a bit weird to use an ItemHeader, might be better to use a spacer or something.

-[Unknown]

Comment (iota97) [reply UI/TouchControlLayoutScreen.cpp:622]: Mhh, I was wondering to use it to add a tip like: "You may edit visibility and add additional controls pressing the Customize button on the left" as that screen might get missed.

Tho' I was not really sure how to write it down and then I forgot :P

**Comment by Gamemulatorer** [comment]
Can you fix the word "customize" is not correct align on my android 11 with 6.5 inches screen.
Screenshot_2021-09-12-17-10-58-282_org ppsspp ppsspp

**Pull request merged**